src/components/Molecules/CtaPrompt/CtaPrompt.component.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file CtaPrompt.component.js
+ * Exports a CTA Prompt component.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import CtaMessage from '../CtaMessage/CtaMessage.component';
+
+const cookiePrefix = 'cta_prompt_';
+
+/**
+ * Component that renders a CTA (Call to action) prompt.
+ */
+export default class CtaPrompt extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    type: CtaMessage.propTypes.type
+  };
+
+  static defaultProps = {
+    children: [],
+    type: null
+  };
+
+  constructor(props) {
```

Comment:
Ditto the comment on the cta message component constructor, this can be replaced with:

```javascript
state = {
  shownMessage: null
}
```